tests/bindings/multi-file-shared.slang

```diff
@@ -7,6 +7,10 @@ PUBLIC float4 use(float  val) { return val; };
 PUBLIC float4 use(float2 val) { return float4(val,0.0,0.0); };
 PUBLIC float4 use(float3 val) { return float4(val,0.0); };
 PUBLIC float4 use(float4 val) { return val; };
+
+#ifdef __SLANG__
+[require(hlsl)]
+#endif
```

Comment:
I am not sure why I have to have this one here. Without this change, this test was failing due to the capability system warning. I suspect that there might be a bug in the capability checking logic.

Reply:
That seems problematic.
Have you tried this with Ariel's pending PR?

Reply:
If this test was not failing before, it implies something is broken, and we should probably fix it before merging.

If Ariel's pending PR fixes this issue, then let's merge this PR after his.

Reply:
I figured why this was causing a trouble.
I had to add `metal` to the following aliases,
```
alias any_target = hlsl | glsl | c | cpp | cuda | spirv;
alias any_textual_target = hlsl | glsl | c | cpp | cuda;
alias any_gfx_target = hlsl | glsl | spirv;
```
I am going to submit my fix shortly.